### ISSUE TITLE
Google Plus API Shutdown

### ISSUE THREAD

**Issue by muneebaahmad** [seejohnrun/google_plus#21]
Should we stop using this gem starting March 7th, 2019? See the article below:

https://developers.google.com/+/api-shutdown

**Comment by seejohnrun** [comment]
Yeah - at that point I'll likely just archive this repo - thanks for the heads up!

**Comment by muneebaahmad** [comment]
For people who are looking for a solution for this, I was able to get around this issue by just making an API call to the following endpoint using `https://www.googleapis.com/oauth2/v2/userinfo`. You can use the same access_token in the `Authorization` header as the one you use for this gem. The response from google will look like the following: 

```
{
    "id": "4236436245",
    "email": "example@gmail.com",
    "verified_email": true,
    "name": "David Beckham",
    "given_name": "David",
    "family_name": "Beckham",
    "link": "https://plus.google.com/4236436245",
    "picture": "https://lh3.googleusercontent.com/-a5rGm7P4sdfasdfg/AAAAAAAAAAI/AAAAAAAAC2c/EHpJrKetbKI/photo.jpg",
    "gender": "male",
    "locale": "en"
}
```

I suggest you try it out using Curl, Postman or Insomnia before starting the implementation.

**Comment by seejohnrun** [comment]
I'll leave this issue around for reference for people, but archiving the repo now. Thanks!
